{{ cookiecutter.dir_name }}/src/android/PythonActivity.java

```diff
@@ -3,9 +3,10 @@
 import android.app.Activity;
 import android.os.Bundle;
 import android.util.Log;
+import android.support.v7.app.AppCompatActivity;
 
 
-public class PythonActivity extends Activity {
+public class PythonActivity extends AppCompatActivity {
```

Comment:
This gives apps the modern look in modern devices ([using the Material design](https://developer.android.com/design/material/index.html)), while keeping backwards compatibility in older ones through fallbacks.
If you inherit straight from Activity, the app has the looks of Android 2 or something.

This also enables you to use the [Action Bar](https://developer.android.com/design/patterns/actionbar.html) (see [tutorial here](https://developer.android.com/training/appbar/index.html)) -- don't know if this would be relevant for Toga, though.